pandas/io/xml.py

```diff
@@ -765,7 +764,7 @@ def read_xml(
         expressions. For more complex XPath, use ``lxml`` which requires
         installation.
 
-    namespaces : dict, optional
+    namespaces : dict, list of dicts, optional
```

Comment:
Again namespaces will not be a list of dicts.

Reply:
Yep removed, forgot this